frontend/packages/knative-plugin/src/components/add/event-sources/ContainerSourceSection.tsx

```diff
@@ -51,7 +51,7 @@ const ContainerSourceSection: React.FC<ContainerSourceSectionProps> = ({ title,
     [setFieldValue],
   );
   return (
-    <FormSection title={title} extraMargin fullWidth={fullWidth}>
+    <FormSection title={title} extraMargin fullWidth={fullWidth} dataTest={`${title}-section`}>
```

Comment:
If you accept the spaces you need to change this here as well
```suggestion
    <FormSection title={title} extraMargin fullWidth={fullWidth} dataTest={`${title} section`}>
```